From 9df06fae54f98f83a97139e8ced9cdeb8ea9f694 Mon Sep 17 00:00:00 2001
From: Allen Xu <allen.xu@freescale.com>
Date: Tue, 10 Apr 2012 16:53:01 +0800
Subject: ENGR00179284-1 support ONFI NAND device on mx6q_arm2_pop board

Add a platform data to indicate whether the board support ONFI nand

Signed-off-by: Allen Xu <allen.xu@freescale.com>
---
 include/linux/mtd/gpmi-nand.h |    1 +
 1 file changed, 1 insertion(+)

diff --git a/include/linux/mtd/gpmi-nand.h b/include/linux/mtd/gpmi-nand.h
index 6659446..e4b52f2 100644
--- a/include/linux/mtd/gpmi-nand.h
+++ b/include/linux/mtd/gpmi-nand.h
@@ -66,5 +66,6 @@ struct gpmi_nand_platform_data {
 	struct		mtd_partition *partitions;
 	unsigned	partition_count;
 	unsigned int	enable_bbt:1;
+	unsigned int	enable_ddr:1;
 };
 #endif
-- 
1.7.9.5

